com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -239,6 +266,55 @@ private bool CheckAnimStateChanged(out int stateHash, out float normalizedTime,
                 }
             }
 
+            // Whether the layer has changed or not, we always check to see if parameters have changed and send updates if they have.
```

Comment:
I think for a future PR we could create a full index of what changed and then send only what has changed.
For this PR I kept it relatively simple since updating meant it would send all values anyway.